In [4]:
from scipy.stats import norm
import scipy.integrate as integrate
import numpy as np

In [2]:
def pdf(d):
    return 20*np.exp(-20*(d-12.5))

In [5]:
integrate.quad(pdf,12.5,12.6)
#quad is for simple integrate

(0.8646647167633718, 9.599706771316849e-15)

##### Here the proportion inbetween 12.5 and 12.6 is 0.86466 and the conclusion we can take is 0.864 is the area covered by pdf inbetween 12.5 to 12.6 i.e. p(12.5<=x<12.6)

Since we dont have pdf for < 12.5. So, for lesser than 12.5 we assume it to be 0. For which CDF=0